Core/clim-core/panes.lisp

```diff
@@ -2383,32 +2382,38 @@ SCROLLER-PANE appear on the ergonomic left hand side, or leave set to
 (defmethod handle-repaint ((pane label-pane) region)
   (declare (ignore region))
   (let* ((region (sheet-region pane))
-         (label (clime:label-pane-label pane))
-         (m0 2)
-         (a (text-style-ascent (pane-text-style pane) pane))
-         (d (text-style-descent (pane-text-style pane) pane))
-         (text-width (text-size pane label)))
+         (align-x (pane-align-x pane))
+         (label (clime:label-pane-label pane)))
     (with-bounding-rectangle* (x1 y1 x2 y2) region
       (multiple-value-bind (ileft itop iright ibottom
-                            text-x-offset text-y-offset
+                            text-offset text-width text-height
                             border-margin)
           (label-pane-margins pane)
-        (declare (ignore ileft itop iright ibottom))
-        (multiple-value-bind (tx ty)
-            (values (ecase (pane-align-x pane)
-                      (:left (+ x1 text-x-offset))
-                      (:right (- x2 text-x-offset text-width))
-                      (:center (- (/ (- x2 x1) 2) (/ text-width 2))))
+        (declare (ignore itop ibottom))
+        (multiple-value-bind (text-pivot-x text-pivot-y dx)
+            (values (ecase align-x
+                      (:left (+ x1 ileft text-offset))
+                      (:right (- x2 iright text-offset))
+                      (:center (/ (- x2 x1) 2)))
                     (ecase (label-pane-label-alignment pane)
-                      (:top (+ y1 text-y-offset))
-                      (:bottom (- y2 m0 d))))
+                      (:top    (+ y1 text-offset))
+                      (:bottom (- y2 text-offset text-height)))
+                    (ecase align-x
+                      (:left 0)
+                      (:right (- text-width))
+                      (:center (- (/ text-width 2)))))
           ;; Draw label.
-          (draw-rectangle* pane x1 (- ty a) x2 (+ ty d) :ink (pane-background pane))
-          (draw-text* pane label tx ty)
+          (draw-rectangle* pane x1 text-pivot-y x2 (+ text-pivot-y text-height)
+                           :ink (pane-background pane))
+          (draw-text* pane label text-pivot-x text-pivot-y
+                      :align-x align-x :align-y :top)
           ;; Draw border around child without drawing over the label
           ;; text.
           (when (sheet-child pane)
-            (let ((text-region (make-rectangle* (- tx m0) (- ty a) (+ tx text-width m0) (+ ty d))))
+            (let* ((text-x (+ text-pivot-x dx))
+                   (text-region (make-rectangle*
+                                 text-x text-pivot-y
+                                 (+ text-x text-width) (+ text-pivot-y text-height))))
```

Comment:
put the first argument in the same line as the function invocation

Reply:
Done.